src/components/__tests__/FlawSearchView.spec.ts

```diff
@@ -99,6 +99,10 @@ describe('FlawSearchView', () => {
     });
   });
 
+  afterEach(() => {
+    vi.clearAllMocks();
+  });
+
   afterAll(() => {
     vi.clearAllMocks();
   });
```

Comment:
What is the reason to clear mocks in both, after each and all? Just curious

Reply:
Without clearing, mock call are not cleared especially for emit and router.replace mock